cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/dataset/MetadataDataset.java

```diff
@@ -711,7 +713,13 @@ private SearchResults searchByDefaultIndex(SearchRequest request) {
         while ((next = scanner.next()) != null) {
           Optional<MetadataEntry> metadataEntry = parseRow(next, column, request.getTypes(),
                                                            request.shouldShowHidden());
-          metadataEntry.ifPresent(results::add);
+          //metadataEntry.ifPresent(results::add);
+          if (metadataEntry.isPresent()) {
+            MetadataEntry e = metadataEntry.get();
```

Comment:
unused `e`, `s` and `m` local variables?

Reply:
[#11489](https://github.com/cdapio/cdap/pull/11489)